api/src/services/FedoraCatalog.ts

```diff
@@ -70,6 +75,30 @@ class FedoraCatalog {
             return { ...acc, ...model.datastreams };
         }, {});
     }
+
+    async getFavoritePids(): Promise<Record<string, string>> {
+        const pids = this.config.favoritePids;
+        const result = {};
+        if (pids.length > 0) {
+            const queryParts = pids.map((pid) => {
+                return `id:"${pid.replace('"', "")}"`;
+            });
+            const query = queryParts.join(" OR ");
+            const solrResponse = await this.solr.query(this.config.solrCore, query, {
+                fl: "id,title",
+                rows: queryParts.length.toString(),
```

Comment:
Isn't this pid.length.toString?

Reply:
Also addressed by d51f3ea7b23d613bca5c7a5b3eb7bb925463450d.